powershell/enterprisePolicies/README.md

```diff
@@ -0,0 +1,150 @@
+# Power Platform Enterprise Policies PowerShell Scripts
+
+These scripts introduces the capabilities of the Enterprise Policies ARM resources. The sample scripts provides capabilities to create, update, delete and get enterprise policies.  
+It also provides capabilities to apply the enterprise policies to environments.
+
+## How to run setup scripts
+
+1. **Install module script** : This script installs the required module to run Enterprise Policies scripts.</br>
+Script name: InstallPowerAppsCmdlets.ps1</br>
+Run the script to import required PowerShell modules.
+
+2. **Setup Azure subscription for Microsoft.PowerPlatform** : This script registers the Azure subscription for Microsoft.PowerPlatform resource provider and also allow lists the subscription for enterprisePoliciesPreview feature.</br>
+Script name : SetupSubscriptionForPowerPlatform.ps1</br>
+Run the script to setup Azure subscription for Microsoft.PowerPlatform
+
+## How to run CMK scripts
+
+The CMK scripts are present in folder Cmk at current location
+
+1. **Create CMK Enterprise Policy** : This script creates a CMK enterprise policy</br>
+Script name : CreateCMKEnterprisePolicy.ps1</br>
+Input parameters :
+    - subscriptionId : The subscriptionId where CMK enterprise policy needs to be created
+    - resourceGroup : The resource group where CMK enterprise policy needs to be created
+    - enterprisePolicyName : The name of the CMK enterprise policy resource
+    - enterprisePolicyLocation : The Azure geo where CMK enterprise policy needs to be created
```

Comment:
Since "geo location" can be confusing between Azure regions and Power Platform geos, let's clarify this. Is the correct terminology here "Azure geo" or "Power Platform geo"? It might also help if we provide some sample values.

Reply:
Enterprise Policy is an ARM resource and the "geo" refers to Azure geos.  I will add a script that will return all the supported locations for Enterprise Policy. The output will be coming directly from manifest so it will have the up to date list of locations supported. The outputs from the script will be as below : 
<img width="122" alt="image" src="https://user-images.githubusercontent.com/80298760/200386386-f879831a-9a3f-494e-bcf4-4a5a1ca57cc5.png">